### PR TITLE
updated the "Assigning permissions..." to new ui

### LINKED ISSUE #1487: Assigning permissions and access using roles and groups chapter
### Description

Update this Server Administration Guide chapter for the new Admin Console

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: keycloak/keycloak#18593

### CODE REVIEW

File: server_admin/topics/roles-groups/con-role-scope-mappings.adoc [hunk 1]
```diff
@@ -8,12 +8,24 @@ On creation of an OIDC access token or SAML assertion, the user role mappings be
...
 
-By default, each client gets all the role mappings of the user. You can view the role mappings in the *Scope* tab of each client.
+By default, each client gets all the role mappings of the user.
+ifeval::[{project_community}==true]
+You can view the role mappings in the *Scope* tab of each client.
+. Click *Clients* in the menu
+. Click the client to go to the details.
+. Click *Client scopes* tab.
+. Click *<client-name>-dedicated* scope.
+. Click *Scope* tab.
```
Comment (andymunro) [inline server_admin/topics/roles-groups/con-role-scope-mappings.adoc]:
```suggestion
. Click the *Scope* tab.
```

File: server_admin/topics/roles-groups/con-role-scope-mappings.adoc [hunk 2]
```diff
@@ -8,12 +8,24 @@ On creation of an OIDC access token or SAML assertion, the user role mappings be
 
 _Role Scope Mappings_ limit the roles declared inside an access token.  When a client requests a user authentication, the access token they receive contains only the role mappings that are explicitly specified for the client's scope.  The result is that you limit the permissions of each individual access token instead of giving the client access to all the users permissions.  
 
-By default, each client gets all the role mappings of the user. You can view the role mappings in the *Scope* tab of each client.
+By default, each client gets all the role mappings of the user.
+ifeval::[{project_community}==true]
+You can view the role mappings in the *Scope* tab of each client.
+. Click *Clients* in the menu
+. Click the client to go to the details.
+. Click *Client scopes* tab.
```
Comment (andymunro) [inline server_admin/topics/roles-groups/con-role-scope-mappings.adoc]:
```suggestion
. Click the *Client scopes* tab.
```

**Comment by andymunro** [comment]
@edewit Please change the chapter title to:  Assigning permissions using roles and groups 
So that it creates the correct anchor ID for when @agagancarczyk creates a PR to address the Learn More issue: https://github.com/keycloak/keycloak-admin-ui/issues/2571
More comments to come.

File: server_admin/topics/roles-groups/proc-creating-realm-roles.adoc [hunk 3]
```diff
@@ -2,7 +2,12 @@
...
 = Creating a realm role
 [role="_abstract"]
+ifeval::[{project_community}==true]
+Realm-level roles are a namespace for defining your roles. To see the list of roles, click *Realm Roles* in the menu.
+endif::[]
+ifeval::[{project_product}==true]
 Realm-level roles are a namespace for defining your roles. To see the list of roles, click *Roles* in the menu.
+endif::[]
 
 image:{project_images}/roles.png[]
```
Comment (andymunro) [inline server_admin/topics/roles-groups/proc-creating-realm-roles.adoc:13]: Step one in the procedure should be

ifeval::[{project_product}==true]
. Click *Add Role*.
endif::[]
ifeval::[{project_community}==true]
. Click *Create Role*.
endif::[]

File: server_admin/topics/roles-groups/proc-converting-composite-roles.adoc [hunk 4]
```diff
@@ -8,9 +8,17 @@ Any realm or client level role can become a _composite role_. A _composite role_
 
 .Procedure
 
+ifeval::[{project_community}==true]
+. Click *Realm Roles* in the menu.
+. Click the role that you want to convert.
+. Click the "Action menu"
+. Click *Add associated roles*
```
Comment (andymunro) [inline server_admin/topics/roles-groups/proc-converting-composite-roles.adoc]:
```suggestion
. From the *Action* list, select *Add associated roles*.
```

File: server_admin/topics/roles-groups/proc-assigning-role-mappings.adoc [hunk 5]
```diff
@@ -8,8 +8,15 @@ You can assign role mappings to a user through the *Role Mappings* tab for that
...
 . Click the *Role Mappings* tab.
+ifeval::[{project_community}==true]
+. Click the *Assign Role* button.
+. Select the role you want to assign to the user from the dialog.
+. Click *Assign*.
+endif::[]
+ifeval::[{project_product}==true]
 . Click the role you want to assign to the user in the *Available Roles* box.
 . Click *Add selected*.
+endif::[]
```
Comment (andymunro) [inline server_admin/topics/roles-groups/proc-assigning-role-mappings.adoc]:
```suggestion
ifeval::[{project_product}==true]
. Click *Users* in the menu.
. Click the user that you want to perform a role mapping on. If the user is not displayed, click *View all users*.
. Click the *Role Mappings* tab.
. Click the role you want to assign to the user in the *Available Roles* box.
. Click *Add selected*.
endif::[]
ifeval::[{project_community}==true]
. Click *Users* in the menu.
. Click the user that you want to perform a role mapping on.
. Click the *Role mappings* tab.
. Click *Assign role*.
. Select the role you want to assign to the user from the dialog.
. Click *Assign*.
endif::[]
```

File: server_admin/topics/roles-groups/proc-managing-groups.adoc [hunk 6]
```diff
@@ -17,10 +17,17 @@ The following example includes a top-level *Sales* group and a child *North Amer
 To add a group:
 
 . Click the group.
+ifeval::[{project_community}==true]
+. Click *Create group*.
+. Enter a group name in the *Create Group* dialog.
```
Comment (andymunro) [inline server_admin/topics/roles-groups/proc-managing-groups.adoc]:
```suggestion
. Enter a group name.
```

File: server_admin/topics/roles-groups/proc-managing-groups.adoc [hunk 7]
```diff
@@ -17,10 +17,17 @@ The following example includes a top-level *Sales* group and a child *North Amer
...
+. Click *Create*.
```
Comment (andymunro) [inline server_admin/topics/roles-groups/proc-managing-groups.adoc:23]: Add one more step.
. Click the group name.

File: server_admin/topics/roles-groups/con-role-scope-mappings.adoc [hunk 8]
```diff
@@ -8,12 +8,24 @@ On creation of an OIDC access token or SAML assertion, the user role mappings be
 
 _Role Scope Mappings_ limit the roles declared inside an access token.  When a client requests a user authentication, the access token they receive contains only the role mappings that are explicitly specified for the client's scope.  The result is that you limit the permissions of each individual access token instead of giving the client access to all the users permissions.  
 
-By default, each client gets all the role mappings of the user. You can view the role mappings in the *Scope* tab of each client.
+By default, each client gets all the role mappings of the user.
+ifeval::[{project_community}==true]
+You can view the role mappings in the *Scope* tab of each client.
```
Comment (andymunro) [inline server_admin/topics/roles-groups/con-role-scope-mappings.adoc]:
```suggestion
You can view the role mappings for a client.

.Procedure
```

File: server_admin/topics/roles-groups/con-role-scope-mappings.adoc [hunk 9]
```diff
@@ -8,12 +8,24 @@ On creation of an OIDC access token or SAML assertion, the user role mappings be
...
+. Click *Clients* in the menu
+. Click the client to go to the details.
+. Click the *Client scopes* tab.
+. Click *<client-name>-dedicated* scope.
```
Comment (andymunro) [inline server_admin/topics/roles-groups/con-role-scope-mappings.adoc]: I think this is supposed to be:
. Click the link under *Assigned client scope*.
 OR the link in the row with
*Dedicated scope and mappers for this client*

File: server_admin/topics/roles-groups/proc-managing-groups.adoc [hunk 10]
```diff
@@ -38,13 +45,24 @@ To add a user to a group:
...
+endif::[]
+ifeval::[{project_product}==true]
 . Select a group from the *Available Groups* tree.
 . Click *Join*.
+endif::[]
 
 To remove a group from a user:
 
+ifeval::[{project_community}==true]
+. Click *Leave* button on the group table row.
```
Comment (andymunro) [inline server_admin/topics/roles-groups/proc-managing-groups.adoc]:
```suggestion
. Click *Users* in the menu.
. Click the user to be removed from the group.
. Click *Leave* on the group table row.
```

File: server_admin/topics/roles-groups/proc-specifying-default-groups.adoc [hunk 11]
```diff
@@ -4,8 +4,15 @@
...
+. Click *Realm settings* in the menu.
+. Click the *User registration* tab.
+. Click the *Default Groups* tab.
+endif::[]
+ifeval::[{project_product}==true]
 . Click *Groups* in the menu.
 . Click the *Default Groups* tab.
+endif::[]
 
 .Default groups
```
Comment (andymunro) [inline server_admin/topics/roles-groups/proc-specifying-default-groups.adoc:17]: Can you recapture this screen after making it smaller.  The text is hard to read.

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks, @edewit.  A few comments to address.

File: server_admin/topics/roles-groups/proc-using-default-roles.adoc [hunk 12]
```diff
@@ -7,8 +7,15 @@ Use default roles to automatically assign user role mappings when a user is crea
 
 .Procedure
 
+ifeval::[{project_community}==true]
+. Click *Realm settings* in the menu
```
Comment (andymunro) [inline server_admin/topics/roles-groups/proc-using-default-roles.adoc]:
```suggestion
. Click *Realm settings* in the menu.
```

File: server_admin/topics/roles-groups/con-role-scope-mappings.adoc [hunk 13]
```diff
@@ -8,12 +8,26 @@ On creation of an OIDC access token or SAML assertion, the user role mappings be
 
 _Role Scope Mappings_ limit the roles declared inside an access token.  When a client requests a user authentication, the access token they receive contains only the role mappings that are explicitly specified for the client's scope.  The result is that you limit the permissions of each individual access token instead of giving the client access to all the users permissions.  
 
-By default, each client gets all the role mappings of the user. You can view the role mappings in the *Scope* tab of each client.
+By default, each client gets all the role mappings of the user.
+ifeval::[{project_community}==true]
+You can view the role mappings for a client.
+
+.Procedure
+. Click *Clients* in the menu
```
Comment (andymunro) [inline server_admin/topics/roles-groups/con-role-scope-mappings.adoc]:
```suggestion
. Click *Clients* in the menu.
```

**Comment by andymunro** [comment]
Thanks, @edewit.   Another one ready to merge, @ssilvert.

**Review by andymunro — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**